The test failures appear unrelated to this PR. Are these tests flaky?

```
FAILED tests/components/netatmo/test_select.py::test_select_schedule_thermostats[pyloop]
FAILED tests/components/google_translate/test_tts.py::test_service_say_german_service[pyloop]
ERROR tests/components/google_translate/test_tts.py::test_service_say_german_config[pyloop]
```

--- a/homeassistant/components/wemo/wemo_device.py
+++ b/homeassistant/components/wemo/wemo_device.py
@@ -84,7 +84,7 @@ async def async_register_device(
         # Temporarily handling all exceptions for #52996 & pywemo/pywemo/issues/276
         # Replace this with `except: PyWeMoException` after upstream has been fixed.
         except Exception:  # pylint: disable=broad-except
-            _LOGGER.warning(
+            _LOGGER.exception(
                 "Failed to enable long press support for device: %s", wemo.name
             )
             device.supports_long_press = False

--- a/homeassistant/components/wemo/wemo_device.py
+++ b/homeassistant/components/wemo/wemo_device.py
@@ -83,7 +83,7 @@ async def async_register_device(
             await hass.async_add_executor_job(wemo.ensure_long_press_virtual_device)
         # Temporarily handling all exceptions for #52996 & pywemo/pywemo/issues/276
         # Replace this with `except: PyWeMoException` after upstream has been fixed.
-        except:  # noqa: E722  pylint: disable=bare-except
+        except Exception:  # pylint: disable=broad-except
             _LOGGER.warning(
                 "Failed to enable long press support for device: %s", wemo.name
             )

